spec/services/openid_connect_attribute_scoper_spec.rb

```diff
@@ -96,6 +96,25 @@
       end
     end
 
+    context 'with profile:name scope' do
+      let(:scope) { 'openid profile:name' }
+
+      it 'includes name attributes' do
+        expect(filtered[:given_name]).to be_present
```

Comment:
Yeah we get all these names from the OpenID connect spec of standard claims: http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims